source/store/actions/CaseActions.js

```diff
@@ -114,11 +110,9 @@ export async function fetchCases(user) {
 
       // eslint-disable-next-line no-restricted-syntax
       for await (const c of response.data.data.attributes.cases) {
-        const { encryptedAnswers } = c.forms[c.currentFormId].answers;
-        if (c?.status.type === 'active:ongoing' && encryptedAnswers) {
+        if (c?.status.type === 'active:ongoing') {
           try {
-            const decryptedAnswers = await decryptWithAesKey(user, encryptedAnswers);
-            c.forms[c.currentFormId].answers = JSON.parse(decryptedAnswers);
+            c.forms[c.currentFormId] = await decryptFormAnswers(user, c.forms[c.currentFormId]);
             cases[c.id] = c;
```

Comment:
not relevant to this pr but I suggest refactoring variable names from 'c' to caseItem for readability reasons :)

Why do we not allow this code style - eslint-disable-next-line no-restricted-syntax?
Is there something that prevent us from discarding this rule in the lint config?

One option to make linting happy is to implement using the reduce method like this:
```
    const userCaseCollection = await caseList.reduce(async (collection, caseItem) => {
      const currentCaseForm = caseItem.forms[caseItem.currentFormId];

      if (caseItem.status.type === 'active:ongoing') {
        caseItem.forms[caseItem.currentFormId] = await decryptFormAnswers(user, currentCaseForm);
      }

      collection[caseItem.id] = caseItem;

      return collection;
    }, {});
```
Yes, there are articles on Stackoverflow suggesting implementing this using ```for of``` for performance reasons. But think the gain in performance is negligible in this scoope in contrast to readability? But I'm just a simple man trying

:)